cross/join/src/main.rs

```diff
@@ -29,8 +29,12 @@ use embedded_time::rate::Extensions;
 use hal::clocks::Clock;
 use hal::pac;
 
+<<<<<<< HEAD:cross/join/src/main.rs
```

Comment:
Looks like a merge conflict snuck in. You'll want to accept HEAD here.

Reply:
oops! acd2976739e98ab8e12f3bfdbafca002c0f1cf45